magenta-lib/src/main/scala/magenta/artifact/ArtifactHelper.scala

```diff
@@ -0,0 +1,30 @@
+package magenta.artifact
+
+import com.amazonaws.services.s3.AmazonS3
+import com.amazonaws.services.s3.model.{ListObjectsV2Request, S3ObjectSummary}
+
+import scala.collection.JavaConverters._
+
+object ArtifactHelper {
+  def listObjects(path: S3Location)(implicit s3Client: AmazonS3) = {
```

Comment:
Done - the Riff-Raff play app now shares the common S3 wrappers in `magenta-lib`